src/index.ts

```diff
@@ -1,2 +1,3 @@
 import PercyAgent from './percy-agent-client/percy-agent'
+export * from './utils/agentjs-filename'
```

Comment:
I'm not 100% sure this is the best way to export this from here, so would appreciate and extra set of checks here, @djones 🙏🏼

Reply:
Yeah, that's what I assumed.

Good idea on testing it with an SDK. Let me do that and come back to you here.